graphql_compiler/compiler/subclass.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017-present Kensho Technologies, LLC.
+from graphql.type.definition import GraphQLObjectType, GraphQLUnionType
+import six
+
+
+def _transitive_closure(graph):
+    """Compute the transitive closure of a graph represented as dict inplace."""
```

Comment:
For functions that mutate input, don't return anything. That will make it obvious to the user that the input has to have gotten mutated.

Also, this sentence structure feels a bit off, especially around the `a graph represented as dict inplace` part. Maybe move the `in-place` bit to earlier in the sentence.